pkg/virt-operator/util/readycheck.go

```diff
@@ -192,3 +196,16 @@ func PodIsCrashLooping(pod *k8sv1.Pod) bool {
 
 	return haveContainersCrashed(pod.Status.ContainerStatuses)
 }
+
+func ComputeHash(template *k8sv1.PodTemplateSpec) string {
+	hasher := sha1.New()
+	hasher.Write([]byte(template.String()))
+	return hex.EncodeToString(hasher.Sum(nil))
+}
+
+func GetPodTemplateHash(annotation map[string]string) string {
```

Comment:
I don't think this function is needed at all. An empty/nil map should return `""` on access of a non-existing key.